char_edit.php

```diff
@@ -31,44 +31,44 @@
             <form class='edit_create_form' action='edit_verify.php' method='POST'>
                 <div>
                     <label>Character name:</label>
```

Comment:
youv added ids to the inputs but the labels don't have for attributes, so either add the fors or remove the ids